examples/provider-components/kustomization.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
+- provider-components-ctlplane.yaml
```

Comment:
shouldn't we also kustomize cert-manager?

Reply:
done. Though I doubt the kustomization is useful for our use-cases